subdomains/blog/_posts/2020-12-21-announcing-volta-100.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Announcing Volta 1.0.0"
+date:   2020-12-16 08:00:00 -0700
+categories: update
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+The Volta team is thrilled to announce that Volta 1.0.0 is now available! **Volta is a fast, reliable manager for JavaScript tools that takes the hassle out of switching versions**, allowing you to focus on your work, not your tools.
+<!--more-->
+
+# What does 1.0.0 mean for Volta?
+
+The 1.0.0 release of Volta means more than, "we implemented some high-impact functionality and finally closed some long-standing issues." From the beginning, Volta was built to empower engineers and engineering teams to build amazing JS applications without having to worry about their environment or tooling. For individual engineers, Volta unobtrusively makes sure your tooling is aligned with the needs of your project, allowing you to stay focused on your work. For engineering teams, Volta allows you to manage your infrastructure decision-making centrally without slowing down individual team members. When the team decides to change versions of Node, for example, that doesn't require explicit effort on the part of each developer. With version 1.0.0, we believe Volta delivers on the promise of making tool management disappear.
+
+# What's new?
```

Comment:
I think the most important reason to talk about what's new is really in service of the points I mentioned above. We could possibly abbreviate this a bit further. But I think it doesn't hurt to keep. Even though the main takeaway of the post is the 1.0 launch, I think it just generally shows pragmatism and forward motion to include some "release notes"y content.

Reply:
That's what I was thinking, it also keeps continuity with the other "Announcing" posts for 0.8.0 and 0.9.0.